examples/testprograms/http/http.star

```diff
@@ -4,10 +4,19 @@ load("env", "MEOW")
 print(MEOW)
 
 def on_http_get():
-    resp, err = catch(lambda: http1.get("https://httpbin3212.org/status/404"))
+    resp, err = http1.get("https://httpbin3212.org/status/404", ak.callopts(catch=True))
```

Comment:
This "trick" of multiple assignment happens to be supported in Python and Starlark, and sort-of in JS/TS but not so nicely. But we can't rely on it. Also, none of these options really feels "intuitive" or "easy to read".

Instead, why not just return a single `result` object that contains optional error information alongside an optional `resp` struct property, e.g. a Boolean `ok` property and/or a string `error` property.